.gitignore

```diff
@@ -21,6 +21,7 @@
 /*.trace.db
 /.babel_cache
 /.env
+/.envrc
```

Comment:
We don't have this file in our repo.
But it is handy to store some ENV variables in it locally and read them using `direnv` command.